### DISCUSSION TITLE
Raj Musix Tamil

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#699] [Q&A]
**Channel Name:** Raj Musix Tamil
**Channel ID:** RajMusixTamil.in
**Country:** IN
**Language:** Tamil
**Website:** https://mhdtvworld.xyz/dc54b070-ca37-4b9b-b654-fbac9e9504e9
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
> Website: https://mhdtvworld.xyz/dc54b070-ca37-4b9b-b654-fbac9e9504e9

Link is broken.

**Comment by Sphinxroot** [comment]
https://mhdtvworld.xyz/live/raj-musix/

https://mhdtvworld.xyz/delta/planet/max.php?id=36135&e=.m3u8

but i cant play it in kodi

**Comment by Niyasmkkv** [comment]
Geo blocked in India.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19714
